cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -21,7 +21,7 @@ function up() {
     prepare_config
 
     # Activate cluster-network-addons-operator if flag is passed
-    if [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+    if [ "$WITH_CNAO" == "true" ]; then
```

Comment:
I recall Roman commenting on the benefit of keeping it namespaced.

Reply:
This is kind of internal var, that I didn't namespaced it.

Reply:
It should probably default somewhere to false then. Strictly speaking. Not that we enforce that in general anywhere ;)